smart_campus/app/models.py

```diff
@@ -26,6 +26,8 @@ def create_user(self, email, password=None):
 
         user = self.model(
             email=self.normalize_email(email),
+            nickname=nickname,
+            role=Role.objects.get(name='User'),
```

Comment:
Did you provide command or setup method to ensure that role "User" exists?

Reply:
The staticmethod insert_roles() in models 'Role' should be executed at setup or change of role.

Reply:
Ok, then make sure you write a script for that.

Reply:
It should be run in 'manage.py shell'
Maybe I will create a custom admin command
Or is there a better way ?
The doc:
https://docs.djangoproject.com/en/1.11/howto/custom-management-commands/#module-django.core.management

The command needed:
```
from app.models import Role

Role.insert_roles()
```

Reply:
Yep, I think it is the most intuitive way.
Great Job!